src/reducers/badgesReducer.js

```diff
@@ -25,15 +25,17 @@ import {
   SET_CONTACTS_BADGES,
   FETCHING_CONTACTS_BADGES,
   STOP_FETCHING_CONTACTS_BADGES,
+  SET_BADGE_AWARD_EVENTS,
 } from 'constants/badgesConstants';
-import type { Badges } from 'models/Badge';
+import type { Badges, BadgeRewardEvent } from 'models/Badge';
 
 export type BadgesReducerState = {
   data: Badges,
   contactsBadges: {
     [contactId: string]: Badges,
   },
   isFetchingBadges: boolean,
+  badgesEvents: BadgeRewardEvent[]
```

Comment:
```suggestion
  badgesEvents: BadgeRewardEvent[],
```

Reply:
🤦
thought I was getting better at not missing those